lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -748,8 +748,8 @@ previewers.git_branch_log = defaulter(function(opts)
     for i = 1, #content do
       local line = content[i]
       local _, hstart = line:find "[%*%s|]*"
+      local _, hend, _ = line:find "[%*%s|]*%s(%w*)"
```

Comment:
```suggestion
      local hstart, hend = line:find "[0-9a-fA-F]+"
```
Do you think this simplification will work? I believe that the commit hash is always the first hex string.

Reply:
Yep that's way better.